<launch>
    <!-- <include file="$(find turtlebot_navigation)/launch/gmapping_demo.launch"/> -->
    <!-- <param name="/move_base/global_costmap/inflation_layer/inflation_radius" value="0.2" />
    <param name="/move_base/local_costmap/inflation_layer/inflation_radius" value="0.2" />
    <param name="/move_base/global_costmap/robot_radius" value="0.2" />
    <param name="/move_base/local_costmap/robot_radius" value="0.2" />
    <param name="/slam_gmapping/minimumScore" value="1000" /> -->

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find final_project)/final.rviz" />
    <!-- <node name="explore" pkg="final_project" type="explore.py" /> -->


</launch>
